src/index.js

```diff
@@ -468,7 +472,12 @@ const sendError = async (response, acceptsJSON, current, handlers, config, spec)
 		return;
 	}
 
-	response.setHeader('Content-Type', 'text/html; charset=utf-8');
+	const headers = await getHeaders(config.headers, current, absolutePath, null);
+
+	response.writeHead(statusCode, Object.assign(headers, {
```

Comment:
Why assign? You are modifying the headers anyway if you do this. Do you want to modify the original object?
Perhaps you want to do:
```Object.assign({}, headers, {...})```

Reply:
Updated!